### DISCUSSION TITLE
Discount information in the payment terms (Skonto)

### DISCUSSION THREAD

**Discussion by s1nn3r** [stephanstapel/ZUGFeRD-csharp#302] [Ideas]
Unfortunately, the library currently does not provide discount information in the payment terms (Skonto).
Would it be possible to add this? It seems there are two additional fields needed: one for the percentage rate and one for the duration until the discount period ends.
Thank you!
Marc

**Comment by stephanstapel** [comment]
Yes, that might be missing to the library. IIRC this goes into:
https://github.com/stephanstapel/ZUGFeRD-csharp/blob/6fa4bece421966355df9b28944798497ce479132/ZUGFeRD/InvoiceDescriptor22CIIWriter.cs#L779
I am happy to review a pull request and can alternatively offer to implement this on a consulting basis.

**Comment by s1nn3r** [comment]
I understand now that this is not defined with its own fields in XRechnung but just added as a special text to the payment terms description (at least for now).
So no change to the library is needed.
Just add "#SKONTO#TAGE=14#PROZENT=3.00#&#xD;&#xA;" as payment terms description works fine.
